src/style.css

```
@@ -1,3 +1,85 @@
+html {
+  --blue-color: #00ACDC;
+  --black-color: #000;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-align: left;
+  line-height: normal;
+  text-transform: uppercase;
```

just add the padding to the header
```suggestion
  text-transform: uppercase;
  padding:n 0 50px;
```